distrho/src/DistrhoDefines.h

```diff
@@ -74,7 +74,7 @@
 #if defined(__GNUC__)
 # define DISTRHO_DEPRECATED __attribute__((deprecated))
 #elif defined(_MSC_VER)
-# define DISTRHO_DEPRECATED __declspec(deprecated)
+# define DISTRHO_DEPRECATED [[deprecated]] // Note: __declspec(deprecated) it not applicable to enum members
```

Comment:
comments placed on macros that are to be used with other stuff in front should use /* */ style of comments rather than //

Reply:
ok, but note that `override` and `final` definitions are formatted the same

Reply:
right, that is my bad.
I will leave them for now as to not generate merge conflicts.